### ISSUE TITLE
different results with the same input

### ISSUE THREAD

**Issue by hainguyenct** [mmp2/megaman#86]
Hello,
I run file example.py in folder examples.
With the same input, e.g

```
X=
[[ 0.44399868  1.58345008 -0.10397256]
 [ 0.89724097  1.05778984 -1.44154121]
 [ 0.8240493   1.13608912 -0.43348191]
 [ 0.41051068  1.85119328 -0.08814421]
 [-0.65903619  0.14207212  0.24788877]
 [ 0.98089687  0.1742586  -0.80547154]
 [-0.55488662  0.04043679  0.16807402]
 [-0.5233528   1.66523969 -1.8521161 ]
 [-0.94192794  1.5563135  -1.33581507]
 [-0.89054316  1.7400243   0.54510124]]
```
I run twice, and there are 2 different resultsresults after the function `spectral.fit_transform(X)`.
for example
Y1=
```
[[ 0.2921505   0.1212865 ]
 [ 0.27392757  0.02372519]
 [ 0.5506981   0.22735989]
 [-0.05899231 -0.14670033]
 [-0.48160085 -0.21859889]
 [ 0.03642044  0.07639132]
 [ 0.46814198 -0.15150964]
 [ 0.23721618 -0.6066226 ]
 [ 0.15258655 -0.33629939]
 [ 0.02977191 -0.5948518 ]]

```
Y2=
```
[[ 0.22423667 -0.37421422]
 [ 0.25427333  0.42450319]
 [ 0.27039709  0.55471119]
 [ 0.07265556 -0.44721124]
 [-0.36659607  0.1178414 ]
 [-0.1421814  -0.14854459]
 [ 0.445397    0.05315703]
 [ 0.60791817 -0.29251698]
 [ 0.14910215  0.21826375]
 [ 0.24877077  0.00868079]]
```
even I set the seed as below:
```
import random
random.seed(1)
spectral = SpectralEmbedding(n_components=n_components, eigen_solver='arpack',
							 geom=geom)
embed_spectral = spectral.fit_transform(X)
print(X)
print(embed_spectral)
```

How do I use megaman so that I will have the same output with one input for different run times?
Thank you

**Comment by yuchaz** [comment]
Thanks for the report, it only happened when using python2. You can quick fix it by converting radius to float, as the example below:

```
radius = 5
adjacency_method = 'cyflann'
adjacency_kwds = {'radius':float(radius)}
affinity_method = 'gaussian'
affinity_kwds = {'radius':float(radius)}
laplacian_method = 'geometric'
laplacian_kwds = {'scaling_epps':float(radius)}
```

This bug is from the division operation in python2. In python2, `/` operator is integer division therefore `4/3` will be `1` rather than `1.33`.

**Comment by hainguyenct** [comment]
Thanks, this error fixed.
